docs/reference/commandline/swarm_leave.md

```diff
@@ -55,4 +55,7 @@ To remove an inactive node, use the [`node rm`](node_rm.md) command instead.
 * [node rm](node_rm.md)
```

Comment:
See the paragraph above; swarm leave doesn't remove the node from the list of nodes; you still need to _remove_ the node there after it has left; i think it's ok to keep